tests/compiler/class-member-function-as-parameter.ts

```diff
@@ -0,0 +1,16 @@
+class C {
+    aa:i32 = 1;
+    callback(a: i32): i32 {
+      return this.aa + a + 3;
+    }
+  }
+  
+  function expectCallback(c1: (arg0: i32) => i32): i32 {
+    return c1(4);
+  }
+  
+  export function fut(): i32 {
+    const c1 = new C();
+    return expectCallback(c1.callback);
+  }
```

Comment:
Could we simplify it?
```suggestion
class A {
    a: i32 = 1;
    fn(): i32 { return this.a; }
}
  
function run(fn: () => i32): i32 {
    return fn();
}
  
export function test(): i32 {
    return run((new A()).fn);
}
```

Reply:
@MaxGraey, Thank you! Yes, the code is formated

Reply:
Also you should update fixtures via `ASC_FEATURES="*" npm run test:compiler -- --create `

Reply:
Er, no, don't do that, that'll add other irrelevant files (noExportRuntime.debug.js and co.) to the diff. #2722 should get all that in order, but anyhow, committing every updated fixture isn't a good idea.